appcheck/firebase-appcheck-debug-testing/src/test/java/com/google/firebase/appcheck/debug/testing/DebugAppCheckTestHelperTest.java

```diff
@@ -47,50 +46,6 @@ public void setUp() {
     initializeFirebaseApp(ApplicationProvider.getApplicationContext(), OTHER_FIREBASE_APP_NAME);
   }
 
-  @Test
```

Comment:
These two tests should not be removed, as this flow of logic still needs to be verified. Can you substitute `PlayIntegrityAppCheckProviderFactory` for `SafetyNetAppCheckProviderFactory` in these tests?